redash/query_runner/athena.py

```diff
@@ -149,7 +149,12 @@ def run_query(self, query, user):
             column_tuples = [(i[0], _TYPE_MAPPINGS.get(i[1], None)) for i in cursor.description]
             columns = self.fetch_columns(column_tuples)
             rows = [dict(zip(([c['name'] for c in columns]), r)) for i, r in enumerate(cursor.fetchall())]
-            data = {'columns': columns, 'rows': rows}
+            qbytes = None
+            try:
+                qbytes = cursor.data_scanned_in_bytes()
+            except AttributeError as e:
+                debug("Athena Direct can't get data_scanned_in_bytes: %s", e)
+            data = { 'columns': columns, 'rows': rows, 'data_scanned': [{ qbytes }] }
```

Comment:
I'm expecting qbytes to return a string. It needs to be saved to the `data_scanned` "key" of the data dictionary. The syntax is odd, but it works and I got to it from a stackoverflow answer tracking down a problem at some point.